app/screens/SupportUs/translations/en.json

```diff
@@ -1,7 +1,7 @@
 {
-	"SUPPORT_US_SCREEN_TITLE": "Support us!",
-	"SUPPORT_US_SCREEN_WHY_DONATE_TITLE": "Why you should consider supporting us",
-	"SUPPORT_US_SCREEN_WHY_DONATE_CONTENT": "NMF.earth is not for profit, we have only raised a small amount on Kickstarter and we keep working a lot on the project even if it's not financially sustainable. Besides that, we have many costs such as servers, domain name, softwares licences, apple developer account etc.",
-	"SUPPORT_US_SCREEN_TITLE_CONSIDER_DONATE": "If you like this project, please consider donating a small amount to help us keep going! And since everything is transparent with us, you will be able to see how we used your donation by looking at end of the year report.",
-	"SUPPORT_US_SCREEN_PLATEFORM": "You can do so on Open Collective or on Patreon."
-}
\ No newline at end of file
+	"SUPPORT_US_SCREEN_TITLE": "Hey you there would you like supporting us!",
+	"SUPPORT_US_SCREEN_WHY_DONATE_TITLE": "Now you may ask why you should consider supporting us",
+	"SUPPORT_US_SCREEN_WHY_DONATE_CONTENT": "See NMF.earth is not for profit, we have only raised a small amount on Kickstarter and we keep working a lot on the project even if it's not financially sustainable. Besides that, we have many costs such as servers, domain name, softwares licences, apple developer account etc.",
+	"SUPPORT_US_SCREEN_TITLE_CONSIDER_DONATE": "So If you like this project, please consider donating a small amount to help us keep going! And since everything is transparent with us, you will be able to see how we used your donation by looking at end of the year report.",
```

Comment:
Starting a sentence by "So" is also not proper english. Plus if shouldn't be in capital